modules/tf-aws-open-next-public-resources/main.tf

```diff
@@ -499,6 +514,145 @@ resource "aws_cloudfront_cache_policy" "cache_policy" {
+          header   = items.value.header
+          override = items.value.override
+          value    = items.value.value
+        }
+      }
+    }
+  }
+
+  dynamic "remove_headers_config" {
+    for_each = length(var.response_headers.remove_headers) > 0 ? [true] : []
+
+    content {
+      dynamic "items" {
+        for_each = var.response_headers.remove_headers
+
+        content {
+          header = items.value
+        }
+      }
+    }
+  }
+
+  dynamic "security_headers_config" {
+    for_each = try(var.response_headers.security_headers_config, null) != null ? [true] : []
+
+    content {
+      dynamic "content_security_policy" {
+        for_each = try(var.response_headers.security_headers_config.content_security_policy, null) != null ? [var.response_headers.security_headers_config.content_security_policy] : []
+
+        content {
+          content_security_policy = content_security_policy.value.policy
+          override                = content_security_policy.value.override
+        }
+      }
+
+      dynamic "content_type_options" {
+        for_each = try(var.response_headers.security_headers_config.content_type_options, null) != null ? [var.response_headers.security_headers_config.content_type_options] : []
+
+        content {
+          override = content_type_options.value.override
+        }
+      }
+
+      dynamic "frame_options" {
+        for_each = try(var.response_headers.security_headers_config.frame_options, null) != null ? [var.response_headers.security_headers_config.frame_options] : []
+        content {
+          frame_option = frame_options.value.frame_option
+          override     = frame_options.value.override
+        }
+      }
+
+      dynamic "referrer_policy" {
+        for_each = try(var.response_headers.security_headers_config.referrer_policy, null) != null ? [var.response_headers.security_headers_config.referrer_policy] : []
+
+        content {
+          referrer_policy = referrer_policy.value.policy
+          override        = referrer_policy.value.override
+        }
+      }
+
+      dynamic "strict_transport_security" {
+        for_each = try(var.response_headers.security_headers_config.strict_transport_security, null) != null ? [var.response_headers.security_headers_config.strict_transport_security] : []
+
+        content {
+          access_control_max_age_sec = strict_transport_security.value.max_age
+          include_subdomains         = try(strict_transport_security.value.include_subdomains, null)
+          override                   = strict_transport_security.value.override
+          preload                    = try(strict_transport_security.value.preload, null)
+        }
+      }
+
+      dynamic "xss_protection" {
+        for_each = try(var.response_headers.security_headers_config.xss_protection, null) != null ? [var.response_headers.security_headers_config.xss_protection] : []
+
+        content {
+          mode_block = try(xss_protection.value.mode_block)
```

Comment:
I don't think the try is necessary, same with line 640 too

Reply:
You are correct, I've removed the unneeded try's